includes/class-webmention-tools.php

```diff
@@ -53,7 +53,39 @@ public static function register_routes() {
 
 	public static function read( $request ) {
 		$url              = $request->get_param( 'url' );
+		$request = new Webmention_Request();
+		$return  = $request->fetch( $url );
 
+		if ( is_wp_error( $return ) ) {
+			return $return;
+		}
+		$json = array();
+		$meta = new Webmention_Handler_Meta();
```

Comment:
can we use this to implement the handler iteration? so that you do not have to initiate all handlers here?